getting_started/scripting/c_sharp/c_sharp_basics.rst

```diff
@@ -24,7 +24,7 @@ A good starting point for checking its capabilities is the `Compatibility <http:
 Setup C# for Godot
 ------------------
 
-To use C# in Godot you must have `Mono <http://www.mono-project.com/download/>`_ installed (at least version 5.2), as well 
+To use C# in Godot you must have `Mono <http://www.mono-project.com/download/>`_ installed, at least version 5.10 on Linux and Windows and 5.8 on MacOS, as well 
```

Comment:
Since 3.0.3 is not stable yet, and 3.0.2 requires *strictly* Mono 5.4 (unless compiled manually), I think things should be made clearer for both versions.